rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/WindupWildcardImportResolver.java

```diff
@@ -65,23 +62,18 @@ public String resolve(JavaType<?> source, String type)
             classNameLookedUp.add(type);
 
             // search every wildcard import for this name
-            Importer<?> importer = (Importer<?>) source;
-            for (Import importDeclaration : importer.getImports())
+            for (String wildcardImport : wildcardImports)
             {
-                if (importDeclaration.isWildcard())
-                {
-                    String wildcardImport = importDeclaration.getQualifiedName();
-                    String candidateQualifiedName = wildcardImport + "." + type;
+                String candidateQualifiedName = wildcardImport + "." + type;
 
-                    JavaClassService javaClassService = new JavaClassService(graphContext);
-                    Iterable<JavaClassModel> models = javaClassService.findAllByProperty(JavaClassModel.QUALIFIED_NAME,
-                                candidateQualifiedName);
-                    if (models.iterator().hasNext())
-                    {
-                        // we found it... put it in the map and return the result
-                        classNameToFQCN.put(type, candidateQualifiedName);
-                        return candidateQualifiedName;
-                    }
+                JavaClassService javaClassService = new JavaClassService(graphContext);
+                Iterable<JavaClassModel> models = javaClassService.findAllByProperty(JavaClassModel.QUALIFIED_NAME,
+                            candidateQualifiedName);
+                if (models.iterator().hasNext())
+                {
+                    // we found it... put it in the map and return the result
```

Comment:
I'm pretty sure that this was there before. I think it can be removed though (no it's not valuable).